driver/sources/src/main/scala/org/virtuslab/ideprobe/IntelliJFixture.scala

```diff
@@ -41,6 +37,11 @@ final case class IntelliJFixture(
     copy(config = newConfig)
   }
 
+  def withPaths(pathsConfig: PathsConfig): IntelliJFixture = {
```

Comment:
PathsConfig is a class that appears in config parsing part of probe, we should't leak it to the normal api. Just accept IdeProbePaths here